src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+# control the java code db initialization (hibernate and seeder)
+app.init-db=false
+spring.jpa.hibernate.ddl-auto=none
```

Comment:
strongly recommend to use spring.jpa.generate-ddl instead of the `spring.jpa.hibernate.ddl-auto`. B/C the first one is generic and the second one is vendor (`hibernate specific`).